M2/Macaulay2/m2/methods.m2

```diff
@@ -277,7 +277,7 @@ length = method(TypicalValue => ZZ, Dispatch => Thing)
 codim = method( Options => true )
 radical = method( Options=>{ Unmixed=>false, CompleteIntersection => null, Strategy => Decompose } )
 regularity = method( TypicalValue => ZZ, Options => { Weights => null } )
-associatedPrimes = method( TypicalValue => List, Options =>{ Strategy => 1 } )
+associatedPrimes = method( TypicalValue => List, Options =>{ Strategy => 1, CodimensionLimit => -1 } )
```

Comment:
You probably didn't intend to make this change.

Reply:
This was leftover from an old commit for PrimaryDecomposition-related code - I'll revert it.